share/ramble/ramble-completion.bash

```diff
@@ -595,6 +595,24 @@ _ramble_repo_rm() {
     fi
 }
 
+_ramble_results() {
+    if $list_options
+    then
+        RAMBLE_COMPREPLY="-h --help"
+    else
+        RAMBLE_COMPREPLY="file_upload"
+    fi
+}
+
+_ramble_results_file_upload() {
+    if $list_options
+    then
+        RAMBLE_COMPREPLY="-h --help"
+    else
+        RAMBLE=""
```

Comment:
This should be `RAMBLE_COMREPLY`. 

It's auto-generated, but you're using an older version of develop which is why it's generated wrong.